checks/libs/wmi/types.py

```diff
@@ -0,0 +1,24 @@
+# stdlib
```

Comment:
I did this to avoid pulling in the sampler which in turn depends on pywintypes. 

Although we mock pywintypes, we do so later, so this it was causing import errors with the tests. I will try to do it more elegantly.